nix/sources.json

```diff
@@ -41,6 +41,18 @@
         "url": "https://github.com/nmattia/niv/archive/82e5cd1ad3c387863f0545d7591512e76ab0fc41.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "nix-filter": {
+        "branch": "master",
+        "description": "a small self-container source filtering lib",
```

Comment:
I guess this should be `s/container/contained/`.

Reply:
Indeed. Ping @zimbatm who maintains nix-filter.